benchmark/Cargo.toml

```diff
@@ -25,6 +25,9 @@ slog-stdlog = "3.0"
 slog-scope = "4.0"
 slog-term = "2.2"
 
+[target.'cfg(not(windows))'.dependencies]
```

Comment:
Why excludes windows?

Reply:
jemallocator doesn't support Windows

Reply:
Should we also exclude MUSL which doesn't support jemalloc?

Reply:
Do we consider MUSL a supported dev platform? I think there are lots of platforms which don't support jemalloc and this is only for benchmarking. I'd be inclined to leave it like this (we need to exclude Windows for CI) and accept a PR if someone is affected by it.